static/script.js

```diff
@@ -1,3 +1,6 @@
-let script = document.createElement("hs-script-loader");
```

Comment:
I think the original code was alright, but this needs to be "script" rather than "hs-script-loader"

Reply:
Yeah, I was it was still not being added in the page source.  document.createElement() should have probably been document.createElement("div"), which is what I'm attempting now.  I'm not super familiar with mdbook so however we can inject the following tracking code that's what I'm aiming for.


<script type="text/javascript" id="hs-script-loader" async defer src="//js.hs-scripts.com/22554992.js"></script>